### ISSUE TITLE
Show list of people who reacted to a message on long press

### ISSUE THREAD

**Issue by kunall17** [zulip/zulip-mobile#1936]

**Comment by jainkuniya** [comment]
(we might to discuss the layout, before starting code for this)

**Comment by borisyankov** [comment]
This is VERY LOW PRIORITY.

**Comment by timabbott** [comment]
Very low priority is a bit stronger than I'd say (because an important part of communication at times is who the 2/5 people who reacted are), but I certainly wouldn't want significant work to go into this before other things that are a decent chunk of work (like being able to add the first reaction at something, or various bug fixes) are done.

**Comment by borisyankov** [comment]
I put it at VERY LOW, because there are several months of work at least that is more important.
Also, it requires a different UI than the one in the web app, plus research on how it is done in competing apps, thus the implementation is not as straightforward.

**Comment by gnprice** [comment]
Closing in favor of #2252 .
